hasher-matcher-actioner/hmalib/lambdas/api/stats.py

```diff
@@ -98,11 +105,14 @@ def default_stats() -> StatResponse:
             metric_time_period,
         )
 
+        total_count = stat_name_to_count[bottle.request.query.stat_name]()
```

Comment:
nit/unsure:`.get(..., <noop>)` instead of `[...]` to avoid key error if we want to send `bottle.request.query.stat_name = "actions"` before it is ready on the backend and not have the API error out?

Reply:
https://github.com/facebook/ThreatExchange/pull/565/files#diff-719551ffa714ce37c8b382114df348cb3c82750380efc186fbe2292c8875b51aR85-R88 would catch this I believe.

Also the error would nudge the dev to implement functionality. Leaving this as is.

Reply:
Ahh, of course github hid that check from me, thanks! 
(links broken for me but figured it was the same file and just went to the line numbers in the url haha)